### DISCUSSION TITLE
How do I sync a folder from my Workspace to my local machine?

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/coder#5780] [General] [answered]
Let's say I have a folder in my Workspace called `/dist`. I want to access that folder on my local machine (Windows 10 PC). I can zip and download from code-server but then I have to do that everytime `/dist` updates (which is frequent).

How can I _sync_ it to a folder on my local machine? 

_Context:_ I'm developing a Chrome extension in my Workspace. To test in the browser, I have to build then upload `/dist` to chrome://extensions. Remote development makes this hard lol but I'm assuming there's an easy way. 

I've heard of `rsync` and `scp` but haven't used them before.

**Comment by matifali** [comment, marked as answer]
You could use one of the following 3rd party tools to achieve data sync.
1. [`rsync`](https://www.digitalocean.com/community/tutorials/how-to-use-rsync-to-sync-local-and-remote-directories)
2. [`unison`](https://github.com/bcpierce00/unison)
3. [`syncthing`](https://github.com/syncthing/syncthing)
